mysql.py

```diff
@@ -410,9 +410,18 @@ def fetch_mysql_db_size(conn):
     		stats[row['db_name']] = row['db_size_mb']
 	return stats
 
+def fetch_mysql_version(conn):
+	result = mysql_query(conn, "select @@version;")
+	return result.fetchone()
+
 def fetch_innodb_os_log_bytes_written(conn):
-	result = mysql_query(conn, "SELECT COUNT from INFORMATION_SCHEMA.INNODB_METRICS where name ='os_log_bytes_written';")
-	stats = result.fetchone()
+	# This feature is only available for mariaDB >= 10.x and MySQL > 5.5.
+	mysql_version = fetch_mysql_version(conn)
```

Comment:
instead of checking version you could handle execption like in other portion of the code with 
except MySQLdb.OperationalError:

Reply:
Done.